examples/osgviewerIPhone/shaders.h

```diff
@@ -0,0 +1,113 @@
+#pragma once 
```

Comment:
I am beginning another review, I've checkout out a branch.  One element I feel needs to be looked at is the need for custom shaders for text.  I have now checked in the text refactor changes into master so the osgText now sets up shaders for GLE2/GLES3 as GL3 core profile. I think we should be able to avoid some of the changes you've made.  Could you try your changes out on master and see if you can roll back the local shader additions.